vite-project/src/components/ListItem.tsx

```diff
@@ -1,11 +1,15 @@
 import "./ListItem.css";
+import { Items } from "../api/api";
+interface Props {
+  item: Items;
+}
 
-function ListItem({ item }) {
+function ListItem({ item }: Props) {
   return (
     <div>
       <div className="list-item">
         <div className="item-images">
-          <img src={item.images} />
+          <img src={item.images?.[0]} />
```

Comment:
ListItem.tsx 파일에서 <img src={item.images} /> 에서 src 가 빨간줄로 'string[]' 형식은 'string' 형식에 할당할 수 없습니다. 라는 오류가 떠서 확인을 해보니 src는 string 하나를 원하는데, item.images가 string[] 이라서 임을 깨달았습니다. 그럼 string[] 을 문자화하는 메소드인 JSON.stringfy(), join() ,toString() 등을 사용하면 되지 않을까 했지만 그렇게하면 유효한 문자열이 될 수 없음을 깨달아서 <img src={item.images?.[0]} /> 이렇게 고쳤습니다만 더 효율적으로 고치는 방법이 있는지 궁금합니다. 그리고 [0] 안에 0을 넣는게 관례상으로 맞는방법인지도 궁금합니다.

---

질문 주신 부분이 여기인 것으로 보여요 😉
의도하신게 이미지의 `0`번째 이미지를 썸네일로 표현하고자 하셨다면 문제 없습니다 !
다만, 기본 이미지도 고려해보시면 좋을 것 같네요 😉
예를 들어서 다음과 같이 해볼 수 있겠어요:
```suggestion
          <img src={item.images?.[0] ?? "/imgs/panda-product-default.jpg"} />
```
여기서 `/imgs/panda-product-default.jpg`는 임의로 작성드렸으며, 실제로 존재하고 유효한 `path`여야 합니다 ☺️